src/components/backend-ai-import-view.ts

```diff
@@ -134,9 +139,9 @@ export default class BackendAIImport extends BackendAIPage {
           --mdc-select-dropdown-icon-color: var(--general-textfield-selected-color);
           --mdc-select-hover-line-color: var(--general-textfield-selected-color);
           --mdc-list-vertical-padding: 5px;
-        }
-        mwc-select.github-select > mwc-list-item {
-          width: 440px; // default width
+          /* Need to be set when fixedMenuPosition attribute is enabled */
+          --mdc-menu-max-width: 440px;
+          --mdc-menu-min-width: 440px;
```

Comment:
Please make the width the same.
<img width="543" alt="image" src="https://user-images.githubusercontent.com/28584164/178412417-9de45c5d-efd7-434b-810f-d39ae23eec0c.png">

Reply:
fixed it.